docs/Using-the-dataset-validator.md

```diff
@@ -1,3 +1,11 @@
+* [Introduction](##introduction)
+* [Running the validator](##running-the-validator)
+* [Offline Validation](##offline-validation)
+* [Validation of non-human data](##validation-of-non-human-data)
+* [Running the validator for multiple studies](##running-the-validator-for-multiple-studies)
```

Comment:
Please change `##` to a single `#`, else it doesn't work.

Reply:
Fixed :)